src/main/java/team/reborn/energy/api/EnergyStorage.java

```diff
@@ -75,6 +76,11 @@ public interface EnergyStorage {
 	 */
 	EnergyStorage EMPTY = EnergyImpl.EMPTY;
 
+	/**
+	 * A {@link DataComponentType} for energy storage.
```

Comment:
The javadoc should mention that this is only for use on a mod's own objects.

For example something like this:
```java
    /**
     * Stock data component type for energy.
     *
     * <p><b>This component should only be used on item stacks from your mod.</b>
     * Otherwise, do not query it or assume it exists.
     * Inter-mod energy interactions should happen using {@link #ITEM}.</b>
     */
```